dns/zone.py

```diff
@@ -1085,3 +1088,47 @@ def from_xfr(xfr, zone_factory=Zone, relativize=True, check_origin=True):
     if check_origin:
         z.check_origin()
     return z
+
+def generate_ptr(zone, relativize=False, zone_factory=Zone):
+    """Generate PTR Zones for a given zone file and returns a dict
+    with origin as key and the zone object as value.
```

Comment:
Please provide more descriptive documentation. The doc string does not tell what exactly is returned and how it can be used. More specifically:
- Does it return one big reverse zone or multiple reverze zones?
- If it returns more than one zone, how records are split between the zones?
- If applicable, is the splitting mechanism generic enough? For IPv4 and IPv6 as well?
- What this function does with non-A/AAAA records in source zone?

All these need to be answered by the docstring. Thank you for understanding.